src/main/java/net/liquidpineapple/pang/objects/ScoreSystem.java

```diff
@@ -35,11 +35,10 @@ public ScoreSystem() {
     }
 
 
-    public void AddScore(int scoreIn) {
+    public static void AddScore(int scoreIn) {
         if(score != 999999999){
```

Comment:
I added:
if(score > 999999999){score = 999999999;}
Should end up exactly right now when it goes over.